protocol/governance/contracts/storage/Election.sol

```diff
@@ -29,6 +29,7 @@ library Election {
         mapping(address => bytes32) ballotIdsByAddress;
         // Number of votes for each candidate
         mapping(address => uint) candidateVotes;
+        // Pointer to current election settings
```

Comment:
Not really a pointer

Btw if new settings are ever added to epoch then suddenly the whole data structure is locked. Should we do an *actual* pointer (ie mapping within another global structure) instead?